assemblies/assembly-rhdh-observability.adoc

```diff
@@ -0,0 +1,12 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="assembly-rhdh-observability"]
+= Enabling observability for {product}
+:context: assembly-rhdh-observability
+
+toc::[]
+
+// Metrics
+{product} provides a `/metrics` endpoint that can be scraped by the Prometheus collector. You can use Prometheus and Grafana to monitor your {product} instance and observe metrics that are scraped from this endpoint.
```

Comment:
```suggestion
```

Do we need to mention Prometheus and Grafana in our product docs which are dedicated to OCP? We are documenting observability via the `ServiceMonitor` construct, not Prometheus and Grafana directly.

Reply:
Updated this in line with what's in the OCP docs and added a link